.github/workflows/report.yml

```diff
@@ -0,0 +1,65 @@
+name: "Test generation of comparison reports"
+
+on:
+  workflow_call:
+    inputs:
+      which-gap:
+        description: 'Which Gap version'
+        default: 'master'
+        required: false
+        type: string
+
+jobs:
+  report:
+    name: "Report"
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Set up Python 3.7"
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
```

Comment:
Can you tell me what error you get exactly, and where, when you remove this? Perhaps point me at a CI log showing the error?

Reply:
I will try to remove it, and see if the error still occurs.

Reply:
No errors, I remember that I used a different ubuntu and not the latest at the beginning, so this was probably the problem.